src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -2916,11 +2918,11 @@ public Mode3D get3DLayout() {
 			case "oulf":
 			case "top-bottom (left eye first)":
 				isAnaglyph = false;
-				return Mode3D.OUL;
+				return Mode3D.ABL;
 			case "ourf":
```

Comment:
@valib Thank you for pointing at this, as i totally missed it.
OK, so we've choose a weird name, as we should have chosen a name similar to FFmpeg nomenclature, IMO.